packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -957,6 +1003,22 @@ class SliverPhysicalParentData extends ParentData {
   /// top left visible corner of the sliver.
   Offset paintOffset = Offset.zero;
 
+  /// The [crossAxisFlex] factor to use for this sliver child.
+  ///
+  /// If null or zero, the child is inflexible and determines its own size.
+  /// If non-zero, the amount of space the child can occupy in the
+  /// cross axis is determined by dividing the free space (after placing the
+  /// inflexible children) according to the flex factors of the flexible
+  /// children.
+  ///
+  /// This value is only used by the [SliverCrossAxisGroup] widget to determine
+  /// how to allocate its [SliverConstraints.crossAxisExtent] to its children.
+  ///
+  /// See also:
+  /// * [SliverCrossAxisGroup], which lays out multiple slivers along the
+  ///   cross axis direction.
+  int? crossAxisFlex = 1;
```

Comment:
Why does this default to 1 again? Sorry I think it has changed a few times and I may be confused. If we are trying to follow the same conventions as flex for boxes, that does not have a default value.

Reply:
Oh you're right, I'll fix this.